lib/cocoapods/target/aggregate_target.rb

```diff
@@ -28,6 +33,18 @@ def product_module_name
       c99ext_identifier(label)
     end
 
+    # @return [Platform] the platform for this target.
+    #
+    def platform
+      @platform ||= target_definition.platform
```

Comment:
I think this is desired, because it's a computed value and it's accessed quite often.